py3status/modules/keyboard_locks.py

```diff
@@ -1,86 +1,69 @@
 # -*- coding: utf-8 -*-
 """
-Monitor CapsLock, NumLock, and ScrLock keys
+Display indicators for CapsLock, NumLock, and ScrLock.
 
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 1)
-    format: display format for this module (default '{caps} {num} {scr}')
-    icon_caps_off: show when Capitals Lock is off (default 'CAPS')
-    icon_caps_on: show when Capitals Lock is on (default 'CAPS')
-    icon_num_off: show when Numeric Lock is off (default 'NUM')
-    icon_num_on: show when Numeric Lock is on (default 'NUM')
-    icon_scr_off: show when Scroll Lock is off (default 'SCR')
-    icon_scr_on: show when Scroll Lock is on (default 'SCR')
```

Comment:
Deprecating things is not my strong suit. Check if the params here exists in the user's config.

Reply:
> Deprecating things is not my strong suit.

Then you need to be very careful about adding things you may later want to remove.

I like the idea of this PR (simplifying things) I'm going to think about how we can make this sort of thing easier to do for users ie nicer formats.